query-languages/m/type-is.md

```diff
@@ -12,7 +12,9 @@ Type.Is(<b>type1</b> as type, <b>type2</b> as type) as logical
 
 ## About
 
-Determines if a value of `type1` is always compatible with `type2`.
+Determines if a value of `type1` is always compatible with `type2`. 
+
+Parameter `type2` should be a primitive type value or a nullable primitive type value. When this requirement is not met, this function's behavior is undefined and so should not be relied on.
```

Comment:
```suggestion
Parameter `type2` should be a primitive (or nullable primitive) type value. Otherwise, this function's behavior is undefined and shouldn't be relied on.
```